solutions/monitoring_platforms_intro/index.asciidoc

```diff
@@ -0,0 +1,131 @@
+--
+* *Collection/ Storage (Monitoring Plane)*
++
+--
+The drop location of the telemetry needs to be determined such as inside the monitoring plane or externally.
+
+Monitoring can result in a large amount of data. Storing such granular data is costly. Therefore an archiving mechanism is required to make sure costs are not exploding. Once archived the ingested telemetry should be removed.
+--
+* *Analysis/ Diagnosis (Monitoring Plane)*
++
+--
+Includes standard problems like:
+
+** Filtering
+** Aggregation
+** Correlation
+** Reformating
+** Comparison against Key Performance Indicatorss (=KPIs). KPIs have no weight in software development unless they are paired with your business goals. You don’t need a handful of KPI metrics for your software team. All you need is the right KPI to help you improve your product or process. KPIs should be SMART (S = Specific; M = Measureable; A = Assignable; R = Realistic; T = Time Bound). Examples: Code Quality KPIs such as Maintainability index, Complexity metrics, Depth of inheritance, Class coupling, Lines of code; Testing Quality such as Test effort, Test coverage; Availability = Mean time between failures, Mean time to recovery/ repair (https://stackify.com/metrics-monitoring-choosing-the-right-kpis/[Link])
+--
+* *Visualization/ Alerting (Monitoring Plane)*
++
+--
+Includes standard problems like:
+
+** Visualization for monitoring consumer
+** Alerts: *Programmatic action* that free the monitoring consumer from manual intervention. It states the trigger and the action to bee executed. One challenging aspect is to minimize the number of alerts or to detect patterns behind multiple alerts. Infering a suitable thresholds can be challing especially if the threshold is not static.
+** Reports
+** Ad-hoc queries
+** Exploration
+--
+* *Improving Feedback Loop (Plane/ Resources)*
++
+--
+Cases where the monitored resources operated outside their baseline should be the starting point for improvements. This might mean a better tuning of alerts and intervention or system requirements.
+--
+
+Integrating and compliance affect the entire pipeline. Telemetry might have to be collected from other systems to achieve a single monitoring plane. However alerts/ notications might have to be forwarded to other systems.
+Of course a monitoring must be compliant regarding the enterprise guidelines.
+
+The following patterns are not dicussed here:
+
+* Provisioning of the monitoring plane and the monitored resourves
+
+The next chapter describes on platform level the monitoring support of various platforms along with concrete solutions for the outlined perspectives.  Monitoring services/ features for specific resources such as containers are out of scope. They are described together with the specific resource type.
+
+=== Azure as Platform
```

Comment:
Wouldn't it be better to move this section to the Azure-specific guide? If I get it right, this document should be an introduction into the topic. So I think this is too specific. Especially if more platforms are added in the future, e.g. AWS. If you leave it like this, you have to add them here as well. This will blow up the document.